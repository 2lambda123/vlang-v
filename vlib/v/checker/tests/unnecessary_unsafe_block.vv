fn main() {
    x := unsafe { 1 }
    assert x == 1
}
