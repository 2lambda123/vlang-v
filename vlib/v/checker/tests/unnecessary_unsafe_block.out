vlib/v/checker/tests/unnecessary_unsafe_block.vv:2:10: notice: unnecessary `unsafe` block
    1 | fn main() {
    2 |     x := unsafe { 1 }
      |          ~~~~~~
    3 |     assert x == 1
    4 | }
